crates/cli/src/subcommands/project/csharp/StdbModule._csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="SpacetimeDB.Runtime" Version="0.10.*" />
+    <PackageReference Include="SpacetimeDB.Runtime" Version="0.11.*" />
```

Comment:
@RReverser I guess we should only bump this when we're about to publish to NuGet, right? If we don't bump this now, will devs using `spacetime init --lang csharp` have issues?

Reply:
They shouldn't. As long as releases are ABI compatible - and I believe 0.11 is - staying on the older version is safe.

Reply:
I think I'm asking the other way around - if we bump this now, then anybody internally using a recently-built `spacetime` binary (off `master`) will generate a project that references nonexistent NuGet packages, right?

Reply:
Yes, sorry, that's what I meant - staying on the older version is _safer_ than switching to the new one too early.

Reply:
Yeah sorry I think I misread.. or something..